roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -2,29 +2,44 @@
 ##Plausible TODOs:
 # Install Helvetica Neue and Consolas from ngi_reports/data/pandoc_templates/assets/fonts/
 
-- name: Create TeXLive destination 
+- stat: path="{{ texLive_dest }}/tlpkg/texlive.profile"
+  register: texlive_installed
+
+- name: Create TeXLive destination
   file: path={{ texLive_dest }} state=directory mode=g+rwxs
   tags: ngi_reports
 
-- name: Download TeXLive
+- name: Download TeXLive (Skipped if installed)
   get_url:
     url: "{{ texLive_url }}"
-    dest: "{{ texLive_dest }}/{{ texLive_file }}"
+    dest: "{{ texLive_dest }}/src/{{ texLive_file }}"
+    when: not texlive_installed
     mode: g+rw
   tags: ngi_reports
 
 - name: Unpack TeXLive
   unarchive:
-    src: "{{ texLive_dest }}/{{ texLive_file }}"
+    src: "{{ texLive_dest }}/src/{{ texLive_file }}"
     dest: "{{ texLive_dest }}"
     copy: no
-    creates: "{{ texLive_path }}/LICENSE"
+    creates: "{{ texLive_dest }}/src/LICENSE"
```

Comment:
Is really the archive unpacked directly under `{{ texLive_dest }}/src/`? Wouldn't it be something like `{{ texLive_dest }}/src/texBlaBla-Foo/LICENSE`?